chapters/001-Routing/1.2-smol-internet/README.md

```diff
@@ -102,6 +102,8 @@ Now we have 2 separate networks. Fantastic! An internet is a group of machines o
 First, hop onto `boudi`:
 
 ```bash
+colima start
```

Comment:
hmmmm.... I'm reluctant to push a specific piece of docker management software. If someone is using docker desktop, their instructions will be different. Maybe we could add a disclaimer? Or a generic instruction to make sure the reader has docker up and running?

Reply:
This was a big stumbling block to getting started for two separate folks. I think we may need a separate section with "How to get started with docker/colima"

Reply:
We had also never told the reader to `restart` to get the VMs running in the first place